test/cli/package-type-alias-export/test.out

```diff
@@ -1,10 +1,7 @@
-__package.rb: Exporting a type alias is not allowed in package specifications https://srb.help/5034
+__package.rb:5: Exporting a type alias is not allowed in package specifications https://srb.help/3722
+     5 |  export MyPackage::A
+          ^^^^^^^^^^^^^^^^^^^
```

Comment:
Two questions:

- Why this limitation?
- What do you think about having an error note that putting the type alias in a module an exporting that module instead if the limitation must remain, so that people don't have to click through to the error reference for quick help?

Reply:
Currently this is an inherited error from the previous implementation. It's not clear to me that we need to carry it forward, but I thought that we could make that decision after landing this PR.

Reply:
I like that suggestion for the error note, that seems super reasonable to me 👍